07LessonSeven/homework/index.html

```diff
@@ -1,10 +1,43 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Profession Portal</title>
   <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="CSS/style.css">
 </head>
 <body>
+  <header>
+    <h1>Profession Portal</h1>
+    <div>
+    <div class="main">
+
+      <div class="a">
+        <ul>
+          <li>Home</li>
+          <li>Companies</li>
+          <li>Positions</li>
+        </ul>
+      </div>
+    </header>
+</section>
+</div>
```

Comment:
There are stray closing tags that don't correspond to an opening tag. Try to focus on finding these extra tags and removing them.